lib/components/overlays/launcher/widgets/app_launcher_button.dart

```diff
@@ -54,9 +55,13 @@ class _AppLauncherButtonState extends State<AppLauncherButton> {
               focusColor: CommonData.of(context).textColor(),
               onTap: () {
                 if (application.systemExecutable == true) {
-                  print(application.runtimeFlags.toString());
+                  if (kDebugMode) {
```

Comment:
@nmcain should we have this enabled in release mode as well for debug purposes?

Reply:
This is a fix for the [`avoid_print`](https://dart-lang.github.io/linter/lints/avoid_print.html) lint warning. 

The alternative is to turn off the lint rule.

Reply:
I wouldn't completely turn it off but as this code is fetching data from the repos, it might be better to turn the rule off for those lines

Reply:
Any other lines you want to keep in release?